lib/rules/no-multiple-empty-lines.js

```diff
@@ -34,6 +46,12 @@ module.exports = function(context) {
                 trimmedLines = lines.map(function(str) {
                     return str.trim();
                 });
+
+            // add the notEmpty lines in there with a placeholder
+            notEmpty.forEach(function(x, i) {
```

Comment:
Yeah, I was kind of thinking the same thing, mostly wasn't fully following the logic below so I didn't want to risk it :) I can take another stab in a few hours